Packs/CommonPlaybooks/Playbooks/playbook-Domain_Enrichment_-_Generic_v2.yml

```diff
@@ -66,6 +66,8 @@ tasks:
             value:
               simple: inputs.Domain
             iscontext: true
+          right:
```

Comment:
This is added automatically by the SDK right?

Reply:
yes, should I remove it?

Reply:
keep it then